app/src/androidTest/java/fr/free/nrw/commons/ViewActions.java

```diff
@@ -0,0 +1,33 @@
+package fr.free.nrw.commons;
+
+import android.view.View;
+
+import androidx.test.espresso.UiController;
+import androidx.test.espresso.ViewAction;
+
+import org.hamcrest.Matcher;
+
+public class ViewActions {
+
+    public static ViewAction clickChildViewWithId(final int id) {
```

Comment:
Is this used by anything?

Reply:
Also maybe better to create new test stuff in Kotlin either way

Reply:
Thanks for pointing it out. Made the change.